### ISSUE TITLE
UT Water Right Data

### ISSUE THREAD

**Issue by rwjam** [WSWCWaterDataExchange/MappingStatesDataToWaDE2.0#80]
https://opendata.gis.utah.gov/datasets/utahDNR::utah-place-of-use

**Comment by rwjam** [comment]
Data imported.

**Comment by rwjam** [comment]
Issue with priority date value in WaDE for UT wr.  They don’t record the date a date datatype, but rather an integer.  That integer value is given as either a) just give a year value YYYY (e.g., 1970), or b) give a full date value YYYYMMDD (e.g., 18611018).  Issue of TOO MANY 1970 values being incorrectly represented in WaDE.

Redownload most current POD and POU data and remap UT wr, be aware of date fields.
- POD: https://opendata.gis.utah.gov/datasets/utahDNR::utah-points-of-diversion/explore?showTable=true
- POU: https://opendata.gis.utah.gov/datasets/utahDNR::utah-points-of-diversion/explore?showTable=true

**Comment by amabdallah** [comment]
Reopening this issue due to updates.
The POU link above is broken now. Here is the list of datasets. 
https://opendata.gis.utah.gov/search?groupIds=4de5aefb5f6540348da4c633bd57cced&q=rights

Do you think the POU layer is now called "[Utah Place of Use Irrigation](https://opendata.gis.utah.gov/maps/utahDNR::utah-place-of-use-irrigation)? The interesting part is that these POUs are only for agriculture and do not include other beneficial uses as in Nevada and Idaho.

**Comment by rwjam** [comment]
Yep.  Looks they changed the name = Utah Place of Use Irrigation.  Contains the same data.  Will update the POU portion of the wr data with this info & update links.

**Comment by amabdallah** [comment]
Jim said:

_That interpretation appears correct. One issue with the surface type is that some springs or drains may have been categorized as surface over the years. The only way to know is to look at the textual description of the source._

![image](https://github.com/WSWCWaterDataExchange/MappingStatesDataToWaDE2.0/assets/3268971/9b4cef42-9f51-469a-8f34-cf81a052e09c)
